liquid-rust-lrir/src/mir/place.rs

```diff
@@ -58,7 +58,7 @@ impl From<Local> for PlaceRef<'_> {
 
 impl<'a> fmt::Display for PlaceRef<'a> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let mut s = format!("{}", self.local);
+        let mut s = format!("{:?}", self.local);
```

Comment:
```suggestion
        let mut s = format!("_{}", self.local.as_usize());
```
Otherwise, places would look weird when printing them?
 `Local(0).0.1.2` vs `_0.0.1.2`

Reply:
I think this is fine because https://doc.rust-lang.org/nightly/nightly-rustc/src/rustc_middle/mir/mod.rs.html#709-715 implements Debug as `_0`.